source/gui/settingsDialogs.py

```diff
@@ -237,17 +237,15 @@ def onOk(self, evt):
 		Sub-classes may extend this method.
 		This base method should always be called to clean up the dialog.
 		"""
-		self.DestroyChildren()
-		self.Destroy()
+		self.DestroyLater()

```

Comment:
Does this destroy the children properly now DestroyChildren is no longer called?

Reply:
Yes, see https://wxpython.org/Phoenix/docs/html/window_deletion_overview.html#automatic-deletion-of-child-windows
> Child windows are deleted from within the parent destructor. This includes any children that are themselves frames or dialogs, so you may wish to close these child frame or dialog windows explicitly from within the parent close handler.